flexible-classroom/develop/supervise-exams-online.mdx

```diff
@@ -0,0 +1,157 @@
+
+
+## Features
+
+|Feature     |Role         |Example |
+|:------------|:------------|:----------|
+|Hardware device detection |Examiner, examinee |Microphone, camera, and speakers of the examiner and examinees are detected|
+|Broadcast to all|Examiner |The examiner initiates an all-hands call to examinees|
+|Examinee real-time monitoring	| Examiner	| Examinees are monitored in real time via their main camera, secondary camera, and screen images |
+|Surveillance video wall | Examiner	| The examiner monitors the examinees' behavior via their video streams presented on the examiner's screen|
+|Examinee video playback | Examiner |The examiner records and plays back video taken via the main camera, secondary camera, and screen images of the examinee |
+|Examiner/AI abnormal exam situation record | Examiner | The examiner *manually* records or the AI *automatically* records the abnormal situation of the exam (you need to implement it yourself)|
+|Screen sharing	|Examinee | Examinees share their screens to prevent cheating. If an examinee turns off screen sharing during the test, the test is suspended|
+|Main camera monitoring | Examinee	| Examinee is monitored via their main device camera |
+|Secondary camera monitoring |	Examinee |	Examinee is monitored via their mobile device camera |
+|Online test questions	| Examinee	| Examinees answer exam questions presented in the classroom |
+
+## How to use online proctoring
+
+The following example uses the <Vg k="FC"/> web demo to illustrate how examinees and examiners use the online
+proctoring feature.
+
+### Examiner role
+
+1. As an examiner, open the <Vg k="FC"/> web demo and click **Create** to create a room.
+
+   ![flexible_classroom_web_demo](/images/flexible-classroom/fc_web_demo.png)
+
+2. Enter the classroom name, class start and end time, then select **Online Proctoring** in **Class mode** and click **Create**.
+
+   ![create_classroom_web_demo](/images/flexible-classroom/create_classroom_web_demo.png)
+
+   You have now successfully created a classroom under the proctoring scenario. In the <Vg k="FC"/> demo,
+   the room creator holds the examiner role by default.
+
+3. Test your hardware before entering the room to ensure that you can monitor the exam.
+
+   ![online_proctoring](/images/flexible-classroom/online_proctoring.png)
+
+   |Hardware   |Purpose                                                      |
+   |:----------|:------------------------------------------------------------|
+   |Camera	   |To broadcast to all examinees and transmit your video        |
+   |Microphone |To broadcast to all candidates and transmit your voice       |
+   |Speaker	   |To monitor the sound of the examinees' environment           |
+
+   Once hardware is successfully tested, click **Next step** to enter the room.
+
+4. After you enter the room, the exam will start automatically at the preset time. You can also click **Start Exam**
```

Comment:
This is not a step, this is an explanation of what happens.

Will: please remove any instance of this word and move to the present tense.